capstone/src/main/webapp/loginAuth.js

```diff
@@ -0,0 +1,12 @@
+function fetchLoginStatus() {
+  fetch('/login').then(response => response.text()).then((loginStatus) => {
+    //Because loginStatus will return either the logout / login message
+    //we can just check the length of return message to ascertain if
+    //we are logged in or out
+    if(loginStatus.length < 100) {
```

Comment:
Could you please explain why the length of the login message will be < 100 if the user is logged in? I think I missed that point.

Reply:
I agree with Carolyn, this logic could be a bit unclear. Maybe have the servlet return 'true' or 'false' and check if the response is 'true' or 'false' or something a bit more deterministic and clear

Reply:
in the login servelet there are a bunch of response.getWriter().println statements. These statements are what is ultimately stored in loginStatus. The total length of these statements for a logged out user is <100, thus we check this length to determine user login/logout status. Pretty convoluted and can definitely be optimized... lol

Reply:
updated to just using loginStatus class which contains login state for better readability